main.js

```diff
@@ -1,42 +1,59 @@
 const collection = document.querySelector('.collection');
-const titleInput = document.querySelector('.title-input');
-const authorInput = document.querySelector('.author-input');
+const title = document.querySelector('.title-input');
+const author = document.querySelector('.author-input');
 const addBtn = document.querySelector('.add-btn');
-let books = [];
+let books = JSON.parse(localStorage.getItem('books')) || [];
 
-function removefromDOM() {
-  const removeBtn = document.querySelectorAll('.remove-btn');
-  removeBtn.forEach((elem, index) => {
-    elem.addEventListener('click', () => {
-      elem.parentNode.remove();
-      books.splice(index, 1);
-      localStorage.setItem('books', JSON.stringify(books));
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  addBooks() {
+    books.push(this);
+  }
```

Comment:
- [ ] You have defined the method `addBooks` inside the Book class but have not used it. Please try to use all the class methods you described in the Book class.
- [ ] You may use the class variable books that I mentioned in the previous comment for adding a new book to the library:
```javascript
lass Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
    this.books.push({ title, author});
  }
```

Reply:
Hi @medaminedev66 🙂 We have a little confusion as to how you suggested we implement the first comment, hoping you don't mind a short zoom call please? 🙏